generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-dialog.component.ts

```diff
@@ -204,7 +204,6 @@ export class <%= entityAngularName %>DialogComponent implements OnInit {
 })
 export class <%= entityAngularName %>PopupComponent implements OnInit, OnDestroy {
 
-    modalRef: NgbModalRef;
```

Comment:
Did you check the cancel button and close icon on the model? I remember this was needed by one of that to work, but it was long time ago may be things changed